### PR TITLE
feat: Use k8s pods instead of jobs

### PR DESCRIPTION
**Pull request by catto**
I made changes for screwdriver-cd/screwdriver#323.
It works well in my environment and leaves no "completed" pods. It doesn't work with the latest version of Screwdriver because of duplication of build Id (screwdriver-cd/screwdriver#399) though.

### CODE REVIEW

**Comment by stjohnjohnson** [comment]
Looks like a good change.  @jer can you review this change as well?

Can you also switch the commit message to be `feat:` because semantic-release will only publish new versions if it's feat or fix (it's a bug).

**Comment by catto** [comment]
@stjohnjohnson Fixed!

File: index.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ class K8sExecutor extends Executor {
      * @param  {String} [options.kubernetes.token]                   API Token (loaded from /var/run/secrets/kubernetes.io/serviceaccount/token if not provided)
      * @param  {String} [options.kubernetes.host=kubernetes.default] Kubernetes hostname
      * @param  {String} [options.kubernetes.serviceAccount=default]  Service Account for builds
-     * @param  {String} [options.kubernetes.jobsNamespace=default]   Jobs namespace
+     * @param  {String} [options.kubernetes.jobsNamespace=default]   Pods namespace for Screwdriver Jobs
```
Comment (jer) [inline index.js]: This should be `options.kubernetes.podsNamespace=default` right?

Comment (stjohnjohnson) [inline index.js]: I think the input is being kept the same to keep backwards compatibility.

**Pull request merged**